backtrace is different for historical reasons (#1009)
ferrocene/ferrocene: ferrocene/doc/internal-procedures/src/subtree-pulls.rst

Sometimes merge conflicts may occur in which case automation opens an issue
describing the conflict and requiring the upstream pull to be done manually.

Backtrace subtree pull
^^^^^^^^^^^^^^^^^^^^^^

> When `rust-lang/backtrace-rs` is updated, the modifications to it's `Cargo.toml`

Reviewer: ```suggestion
When `rust-lang/backtrace-rs` is updated, the modifications to its `Cargo.toml`
```

> When `rust-lang/backtrace-rs` is updated, the modifications to its `Cargo.toml`
> file must be done in `std/cargo.toml` instead, where a comment marks the start of
> the `backtrace` crate dependencies.

Reviewer 2: ```suggestion
When `rust-lang/backtrace-rs` is updated, the modifications to its `Cargo.toml`
file must be copied over to the relevant section in `library/std/Cargo.toml` instead,
where a comment marks the start of the `backtrace` crate dependencies.

This is due to `std` including the sources of backtrace-rs directly, instead of
depending on it as a cargo cargo dependency.
```

Author: So for clarity, the changes should be in both `cargo.toml` ?

> In addition, `<new-crate>/rustc-dep-of-std` must be added to the `backtrace` array in
> `[features]`.

Reviewer 2: ```suggestion
In addition, for any new dependency added this way, `<new-crate>/rustc-dep-of-std` must be
added to the `backtrace` array in `[features]`.
```

Sometimes merge conflicts may occur in which case automation opens an issue
describing the conflict and requiring the upstream pull to be done manually.

Backtrace subtree pull
^^^^^^^^^^^^^^^^^^^^^^

> When `rust-lang/backtrace-rs` is updated, the modifications to its `Cargo.toml`

Reviewer 3: rST `<code>` markup is done with double backtick, not single

Reviewer 3: alright, needs fixing in a future pr then

Author: Probably for the whole Ferrocene docs 👍🏾
